best-practices/README.md

```diff
@@ -218,7 +218,7 @@ Sass
 Browsers
 --------
 
-* Don't support versions of Internet Explorer before IE10.
+* Avoid supporting versions of Internet Explorer before IE10.
```

Comment:
I like the `avoid`, but here's a suggestion:

`* Be conservative about support for versions Internet Explorer pre-IE10, erring on the side of not-supporting without a compelling business reason"